File: Medium/Python 3/Remove_Duplicates_from_Sorted_Array_II(80).py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i = 0
        while (i < len(nums)):
            try:
                if nums[i] == nums[i + 1]:
                    if nums[i] == nums[i + 2]:
                        del nums[i + 2]
                    else:
                        i += 2
                else:
                    i += 1
            except:
                break
                
        return len(nums)

File: Medium/Java/Remove_Duplicates_from_Sorted_Array_II(80).java
class Solution
{
	public int removeDuplicates(int[] A)
	{
		int i = 0;
		int K = 2;								// Duplicates will appear at most 2 times
		
		for(int n : A)
		{
			if(i < K || n > A[i - K])
			{
				A[i++] = n;
			}	
		}
		return i;
	}
}